lib/logger-controller.ts

```diff
@@ -0,0 +1,52 @@
+import { Logger } from "tslog";
+
+export class LoggerController {
+  private logger: Logger;
+  private fakeLogger: FakeLogger = new FakeLogger();
+  private on: boolean = true;
+  constructor(logger: Logger) {
+    this.logger = logger;
+  }
+  public exec(): Logger | FakeLogger {
```

Comment:
Sorry I am not sure we really need this function to leave some logs, but I guess we can make `LoggerController` to a sort of wrapper for the `logger`. Please take a look at below.

```
class LoggerWrapper {
  private logger: Logger;
  constructor(logger: Logger) {
    this.logger = logger;
  }

  public debug(...args: unknown[]): void {
    if (this.isOn) {
      this.logger.debug(...);
    } else {
      // do nothing
    }
}

```

if we do like the above, then we don't need `FakeLogger`, don't we?

Reply:
comment is reflected : https://github.com/line/line-blockchain-developers-sdk-js/pull/86/commits/6de7cf0c660b7f4260c1c1456ec4eb2b22a899b3